### PR TITLE
add: remix icon

### LINKED ISSUE #1791: [Add] Remix Icon
### Site URL

https://remixicon.com/

### Description of the above provided site

Remix Icon is a website that provides Simply Delightful Icons free for both personal and commercial use.

### Category

Frontend

### Sub Category

icons

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [ ] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by akshayitzme**
## Screenshots
image

fixes #1791

### CODE REVIEW

**Comment by CBID2** [comment]
Hi @akshayitzme! Welcome to LinksHub! :) We don't accept spam pull requests, so I'm closing this pull request. Once you've raised an issue, @rupali-codes, @Anmol-Baranwal, or I will assess it. Once you get assigned, follow the directions in your PR form to fill it out adequately. If you need help with this check out the blog post,[Writing Your First Pull Request: Tips, Best Practices, and AI-Powered Tools for Success](https://dev.to/opensauced/writing-your-first-pull-request-tips-best-practices-and-ai-powered-tools-for-success-3bg9).   In the future, refer to our [Contributing Guide](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md) to learn how to effectively contribute to the project! :)

**Comment by akshayitzme** [comment]
> Hi @akshayitzme! Welcome to LinksHub! :) We don't accept spam pull requests, so I'm closing this pull request. Once you've raised an issue, @rupali-codes, @Anmol-Baranwal, or I will assess it. Once you get assigned, follow the directions in your PR form to fill it out adequately and tag one of us to reopen your PR. If you need help with this check out the blog post,[Writing Your First Pull Request: Tips, Best Practices, and AI-Powered Tools for Success](https://dev.to/opensauced/writing-your-first-pull-request-tips-best-practices-and-ai-powered-tools-for-success-3bg9). In the future, refer to our [Contributing Guide](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md) to learn how to effectively contribute to the project! :)

I'm sorry, I understood. btw, aggressively closing a PR is the worst way I have seen to encourage new community contributors. You could've put a sufficiently explaining label or put the PR into draft until the PR owner creates an issue and gets approved by the maintainers.

**Comment by CBID2** [comment]
> > Hi @akshayitzme! Welcome to LinksHub! :) We don't accept spam pull requests, so I'm closing this pull request. Once you've raised an issue, @rupali-codes, @Anmol-Baranwal, or I will assess it. Once you get assigned, follow the directions in your PR form to fill it out adequately and tag one of us to reopen your PR. If you need help with this check out the blog post,[Writing Your First Pull Request: Tips, Best Practices, and AI-Powered Tools for Success](https://dev.to/opensauced/writing-your-first-pull-request-tips-best-practices-and-ai-powered-tools-for-success-3bg9). In the future, refer to our [Contributing Guide](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md) to learn how to effectively contribute to the project! :)
> 
> I'm sorry, I understood. btw, aggressively closing a PR is the worst way I have seen to encourage new community contributors. You could've put a sufficiently explaining label or put the PR into draft until the PR owner creates an issue and gets approved by the maintainers.

Thanks for the feedback @akshayitzme. I’ll definitely keep this in mind as new contributors come in.

**Comment by akshayitzme** [comment]
fixes #1791

File: database/frontend/icons.json
Comment (CBID2) [inline database/frontend/icons.json:207]: [proofreading] 
```suggestion
    "description": "Remix Icon is a website that provides free icons for both personal and commercial use",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
@akshayitzme, I pointed out a proofreading error in my inline comment. In the future, consider running your descriptions through Grammarly or spellcheck before making a pull request. Other than that, you're off to a great start! :)

**Comment by akshayitzme** [comment]
> @akshayitzme, I pointed out a proofreading error in my inline comment. In the future, consider running your descriptions through Grammarly or spellcheck before making a pull request. Other than that, you're off to a great start! :)

Hi @CBID2 , I've committed the proposed changes.

**Review by CBID2 — APPROVED** [review]
Great job @akshayitzme! :) @Anmol-Baranwal, you’re up! :)

**Comment by Anmol-Baranwal** [comment]
> I'm sorry, I understood. btw, aggressively closing a PR is the worst way I have seen to encourage new community contributors. You could've put a sufficiently explaining label or put the PR into draft until the PR owner creates an issue and gets approved by the maintainers.

We are always working to improve new contributors so that they can be ready for their open source journey.

I've seen people mark these PRs as spam, and you would ultimately get disqualified from hacktoberfest.

It is always recommended to ready contributing guidelines, and it is clearly mentioned on how to approach.

> guidelines
image

So, please be patient and follow the guidelines. Creating the issue after submitting the PR is not the recommended way to approach solving an issue. Additionally, draft PRs should be created before the final PR submission.

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me. 

> great resource

**Comment by CBID2** [comment]
> > > I'm sorry, I understood. btw, aggressively closing a PR is the worst way I have seen to encourage new community contributors. You could've put a sufficiently explaining label or put the PR into draft until the PR owner creates an issue and gets approved by the maintainers.
> 
> > 
> 
> > We are always working to improve new contributors so that they can be ready for their open source journey.
> 
> > 
> 
> > I've seen people mark these PRs as spam, and you would ultimately get disqualified from hacktoberfest.
> 
> > 
> 
> > It is always recommended to ready contributing guidelines, and it is clearly mentioned on how to approach.
> 
> > 
> 
> > > guidelines
> 
> > > image
> 
> > 
> 
> > So, please be patient and follow the guidelines. Creating the issue after submitting the PR is not the recommended way to approach solving an issue. Additionally, draft PRs should be created before the final PR submission.
> 
> 
> 
> Yeah, I agree. well, mam, I'm not here for the Hacktoberfest PRs. I randomly saw and loved this project. It's a shame that I didn't find this earlier.
> 
> 
> 
>  It is beautifully designed :sparkles:  and has so many resources most people aren't aware of. Great job by all of the contributors :rocket: . looking forward to contributing more!

Great looking forward to it @akshayitzme! :)

**Pull request merged**
